packages/amplify-provider-awscloudformation/src/transform-graphql-schema.ts

```diff
@@ -143,8 +143,7 @@ function getTransformerFactory(context, resourceDir, authConfig?) {
     let amplifyAdminEnabled: boolean = false;
 
     try {
-      const amplifyMeta = stateManager.getMeta();
-      const appId = amplifyMeta.providers[providerName].AmplifyAppId;
+      const appId = resolveAppId(context);
```

Comment:
if the other call to resolveAppId needs a try, does this also need one?

Reply:
It's in a try/catch already, but there's no need to call resolveAppId here anyways so going to revert.